_assets/mailserver/start.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+MAILSERVER_IMAGE="${MAILSERVER_IMAGE:-statusteam/status-go:0.23.0-beta.6}"
+DEFAULT_HOST_DATA_DIR="$HOME/.status-node"
+HOST_DATA_DIR="${HOST_DATA_DIR:-"${DEFAULT_HOST_DATA_DIR}"}"
+DOCKER_CONTAINER="${DOCKER_CONTAINER:-status-mailserver-node}"
+EXTERNAL_ADDRESS=${EXTERNAL_ADDRESS:-}
+
+if [ ! -x "$(command -v docker)" ]; then
+  echo "Please install docker. For instructions refer to https://docs.docker.com/install/"
+  exit 1
+fi
+
+echo "Create directory ${HOST_DATA_DIR}."
+mkdir -p $HOST_DATA_DIR
+rm -f "$HOST_DATA_DIR/config.json"
+echo "Create configuration file: ${HOST_DATA_DIR}/config.json."
+cat <<EOF >> "$HOST_DATA_DIR/config.json"
```

Comment:
Why not use the config in [`/config/cli/fleet-eth.beta.json`](/config/cli/fleet-eth.beta.json)?

Reply:
I thought that it would be better if someone can run this script without checking out whole repository.